class.jetpack.php

```diff
@@ -1098,8 +1098,10 @@ function user_role_change( $user_id ) {
 	 * Loads the currently active modules.
 	 */
 	public static function load_modules() {
-		if( !self::is_active() && !self::is_development_mode() ) {
-			return;
+		if ( ! self::is_active() && !self::is_development_mode() ) {
+			if ( ! is_multisite() || ! get_site_option( 'jetpack_protect_active' ) ) {
```

Comment:
What is going to happen when this function is called on single sites during `plugin_initialize()`?